wasm-calculator/README.md

```
@@ -0,0 +1,33 @@
+Rust製の計算ロジックをWebAssemblyにコンパイルし、ブラウザ上で実行する簡易計算機です。サーバーを立てずに`index.html`を開くだけで動作します。
+
+## ディレクトリ構成
+
+- `calc/` - 計算ロジックを提供するRustライブラリクレート
+- `index.html` - ブラウザ用のUI
+- `main.js` - WebAssemblyとUIをつなぐJavaScript
+
+## ビルド方法
+
+1. Rustに`wasm32-unknown-unknown`ターゲットと`wasm-bindgen-cli`、`wasm-pack`が必要です。環境に応じて以下を実行してください。
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)